doc/xsd/tester_config.xsd

```diff
@@ -194,4 +203,21 @@
         </xsd:attribute>
     </xsd:complexType>
 
+    <xsd:complexType name="SyntaxOptions">
+        <xsd:annotation>
+            <xsd:documentation>
+                Type of element to specify options on how to handle xml syntax.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:choice minOccurs="0" maxOccurs="unbounded">
+            <xsd:element name="markdown" type="xsd:string">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        Enable or disable strip indentation.
```

Comment:
IMHO the description sounds really surprising vs element name.
I don't understand why stripping is hidden behind markdown name.
Should I add markdown="true" for env if I'd like to strip indentation there? It sounds weird.

I don't mind to add markdown option as well and says that it just enables indent strip for now (but it should be done in a separate patch if we really need the option).

Reply:
markdown was remaned to strip_indent